src/components/galleryHeader/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { FC } from 'react';
+
+import { I18nContext } from '@smg-automotive/i18n-pkg';
+
+import { Language } from 'src/types/language';
+import TranslationProvider from 'src/components/translationProvider';
+import SimpleGrid from 'src/components/simpleGrid';
+import Show from 'src/components/show';
+import Link from 'src/components/link';
+import { CloseIcon } from 'src/components/icons';
+
+import Box from 'src/components/box';
+
+interface GalleryHeaderProps {
+  currentSlide: number;
+  slidesCount: number;
+  onClose: () => void;
+  titleComponent: () => React.ReactNode;
+  language: Language;
+  businessLinkComponent?: () => React.ReactNode;
```

Comment:
as commented here: https://github.com/smg-automotive/listings-web/pull/870#discussion_r1190687082
You might want to consider keeping this in the project and use children for the title component. Non-blocking though

Reply:
Yes, thank you